js/defaults_dialog.js

```diff
@@ -718,7 +867,12 @@ helper.defaultsDialog = (function () {
                 GUI.tab_switch_cleanup(function () {
                     MSP.send_message(MSPCodes.MSP_SET_REBOOT, false, false, function () {
                         //noinspection JSUnresolvedVariable
-                        savingDefaultsModal.close();
+                        if (savingDefaultsModal) {
+                            savingDefaultsModal.close();
+                        }
+                        if (resettingPIDsModal) {
```

Comment:
This shooul;d be checked if exists with `typeof resettingPIDsModal !== 'undefined'`